docs/distributions/nixos/installation.md

```diff
@@ -23,9 +23,13 @@ Note that you may not be able to complete some steps like [Partitioning](#partit
 After booting to the live environment, partition the disk (usually `/dev/nvme0n1`) using `cfdisk` or other tools of your preference.
 If you have followed the pre-installation guide, there should be another partition created just for Linux.
 
-- Remove that and allocate new partitions accordingly - there are no hard rules. Just make sure to not delete the EFI partition and the macOS partition. See the [NixOS Installation Manual](https://nixos.org/manual/nixos/stable/#sec-installation-manual-partitioning-UEFI) for examples.
-- Format the newly provisioned partitions with `mkfs.<filesystem>` commands and `mkswap` a swap partition (if any).
-- Mount the partitions under `/mnt`, and create directories as necessary. Then, enable the swap partition by using `swapon`. You should finally have something that looks similar to this:
+1. List the current partitions, e.g. with `fdisk -l` or `lsblk`. (If you have created a new partition with diskutil on macOS, in normal circumstances, you will see 3 partitions under `nvme0n1`.)
```

Comment:
"If you have created a new partition with diskutil on macOS"

This line isn't needed since the previous step says follow the Pre-Installation guide, which asks to user to use disk utility. So remove this bit from all the steps.